lntest/harness_node.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/lightningnetwork/lnd/lnrpc"
 	"github.com/lightningnetwork/lnd/lnrpc/chainrpc"
 	"github.com/lightningnetwork/lnd/lnrpc/invoicesrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/neutrinorpc"
```

Comment:
git message needs to be in the format `package: description`. So this can be changed into `itest: add neutrino client to harness node`. And same for others.